Error CS1705 when using OpenCvSharp3-AnyCPU

# Summary of your issue
I get the following error when I include package OpenCvSharp3-AnyCPU

```error CS1705: Assembly 'OpenCvSharp.Extensions' with identity 'OpenCvSharp.Extensions, Version=1.0.0.0, Culture=neutral, PublicKeyToken=6adad1e807fea099' uses 'System.Drawing.Common, Version=4.0.0.1, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51' which has a higher version than referenced assembly 'System.Drawing.Common' with identity 'System.Drawing.Common, Version=4.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51'```

# Environment

I am using OpenCVSharp3-AnyCPU (v4.0.0.20181129) in a DLL project that targets both .NET Framework 4.0 and .NET Standard 2.0. The project builds fine for .NET Framework 4.0 but throws the above error when building for .NET Standard 2.0.

--- a/nuget/OpenCvSharp4.Windows.nuspec
+++ b/nuget/OpenCvSharp4.Windows.nuspec
@@ -2,7 +2,7 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
     <metadata>
         <id>OpenCvSharp4.Windows</id>
-        <version>4.0.0.20181225</version>
+        <version>4.0.0.20181228</version>
         <title>OpenCvSharp NuGet package for x64/x86 Windows (same as OpenCvSharp3-AnyCPU)</title>
         <authors>shimat</authors>
         <licenseUrl>http://opensource.org/licenses/BSD-3-Clause</licenseUrl>
@@ -16,19 +16,19 @@
         <tags>Image Processing OpenCV Wrapper FFI opencvsharp</tags>
         <dependencies>
             <group targetFramework="net20">
-                <dependency id="OpenCvSharp4" version="4.0.0.20181225" />
+                <dependency id="OpenCvSharp4" version="4.0.0.20181228" />
                 <dependency id="OpenCvSharp4.runtime.win" version="4.0.0.20181225" />
             </group>
             <group targetFramework="net40">
-                <dependency id="OpenCvSharp4" version="4.0.0.20181225" />
+                <dependency id="OpenCvSharp4" version="4.0.0.20181228" />
                 <dependency id="OpenCvSharp4.runtime.win" version="4.0.0.20181225" />
             </group>
             <group targetFramework="net461">
-                <dependency id="OpenCvSharp4" version="4.0.0.20181225" />
+                <dependency id="OpenCvSharp4" version="4.0.0.20181228" />
                 <dependency id="OpenCvSharp4.runtime.win" version="4.0.0.20181225" />
             </group>
             <group targetFramework="netstandard2.0">
-                <dependency id="OpenCvSharp4" version="4.0.0.20181225" />
+                <dependency id="OpenCvSharp4" version="4.0.0.20181228" />
                 <dependency id="OpenCvSharp4.runtime.win" version="4.0.0.20181225" />
             </group>
         </dependencies>

--- a/nuget/OpenCvSharp4.nuspec
+++ b/nuget/OpenCvSharp4.nuspec
@@ -2,7 +2,7 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
     <metadata>
         <id>OpenCvSharp4</id>
-        <version>4.0.0.20181225</version>
+        <version>4.0.0.20181228</version>
         <title>OpenCvSharp core library. A package of separate native bindings for your OS is required.</title>
         <authors>shimat</authors>
         <licenseUrl>http://opensource.org/licenses/BSD-3-Clause</licenseUrl>
@@ -19,7 +19,7 @@
             <group targetFramework="net40" />
             <group targetFramework="net461" />
             <group targetFramework="netstandard2.0">
-                <dependency id="System.Drawing.Common" version="4.5.0" />
+                <dependency id="System.Drawing.Common" version="4.5.1" />
             </group>
         </dependencies>
         <frameworkAssemblies>

